src/main/kotlin/no/nav/familie/ef/søknad/søknad/SøknadService.kt

```diff
@@ -73,6 +75,17 @@ class SøknadService(
         return KvitteringMapper.mapTilEkstern(kvittering, innsendingMottatt)
     }
 
+    fun sendInnSøknadskvitteringArbeidssøker(
+        arbeidssøker: Arbeidssøker,
+        fnr: String,
+        navn: String,
+        innsendingMottatt: LocalDateTime,
+    ): Kvittering {
+        val søknadDto = SkjemaMapper.mapTilKontrakt(arbeidssøker, fnr, navn, innsendingMottatt)
+        val kvittering = mottakClient.sendInnSøknadskvitteringArbeidssøker(søknadDto)
```

Comment:
rename til å droppe `s`-en slik vi har gjort i flere filer? Og hva med `arbeidsøker` vs `arbeidssøker`?
```suggestion
        val kvittering = mottakClient.sendInnSøknadkvitteringArbeidsøker(søknadDto)
```

Reply:
Vi har jo med 's' flere andre steder, blant annet i hele familie-ef-mottak

Reply:
Det ser ut som at vi burde forholde oss til `arbeidssøker` fremfor `arbeidsøker` basert på kode ellers